website/content/en/preview/troubleshooting.md

```diff
@@ -416,12 +416,12 @@ To correct the problem if it occurs, you can use the approach that AWS EBS uses,
         "Action": [
             "kms:Encrypt",
             "kms:Decrypt",
-            "kms:ReEncrypt",
+            "kms:ReEncrypt*",
```

Comment:
```suggestion
            "kms:ReEncrypt",
```

Is this `*` supposed to be here?

Reply:
Yea @jonathan-innis - Figured it is needed there as well , as per the AWS guide :

https://docs.aws.amazon.com/en_us/autoscaling/ec2/userguide/key-policy-requirements-EBS-encryption.html